source/guides/administration.rst

```diff
@@ -66,7 +66,7 @@ Workspace Management
     Statistics </manage/cloud-reporting>
     User Satisfaction Surveys </manage/cloud-user-satisfaction-surveys>
     Managing Team and Channel Members </manage/cloud-team-and-channel>
-    Cloud Data Export </manage/cloud-data-export>
+    Workspace Migration </manage/cloud-data-export>
 
 * :doc:`Mattermost Cloud Billing </manage/cloud-billing>` - Set up and manage billing for your Mattermost Cloud workspace.
```

Comment:
I think you'll also need a link to the updated page in this section too: * :doc:`Workspace Migration </manage/cloud-data-export>` - Learn how to ....